assets/javascripts/discourse/components/chat-live-pane.js

```diff
@@ -908,14 +911,28 @@ export default Component.extend({
 
   @action
   _setDraftForChannel(draft) {
+    if (draft?.replyToMsg) {
+      draft.replyToMsg = {
+        id: draft.replyToMsg.id,
+        excerpt: draft.replyToMsg.excerpt,
+        user: draft.replyToMsg.user,
+      };
```

Comment:
Here we don't care to store ALL the chat message data, just what we need to render the reply-to.